package.json

```diff
@@ -114,44 +114,43 @@
     "@types/js-yaml": "3.5.29",
     "@types/lodash": "4.14.58",
     "@types/mime": "0.0.29",
-    "@types/node": "7.0.12",
+    "@types/node": "^9.6.2",
     "@types/office-js": "0.0.47",
     "@types/request": "0.0.42",
     "@types/serve-static": "1.7.31",
     "@types/shelljs": "0.7.0",
-    "@types/webpack": "2.2.12",
-    "@types/webpack-dev-server": "1.12.6",
+    "@types/webpack": "^4.1.3",
+    "@types/webpack-dev-server": "^2.9.4",
     "assets-webpack-plugin": "3.5.1",
     "autoprefixer": "6.7.7",
-    "awesome-typescript-loader": "3.1.2",
-    "browser-sync": "2.18.8",
-    "browser-sync-webpack-plugin": "1.1.4",
+    "awesome-typescript-loader": "^5.0.0",
     "chalk": "1.1.3",
-    "copy-webpack-plugin": "4.0.1",
+    "copy-webpack-plugin": "^4.5.1",
     "cpx": "1.5.0",
-    "css-loader": "0.27.3",
-    "extract-loader": "0.1.0",
-    "extract-text-webpack-plugin": "2.1.0",
-    "file-loader": "0.10.1",
-    "html-loader": "0.4.5",
-    "html-webpack-plugin": "2.28.0",
-    "node-sass": "4.5.1",
+    "css-loader": "^0.28.11",
+    "extract-loader": "^2.0.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "file-loader": "^1.1.11",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "node-sass": "^4.8.3",
     "nodemon": "^1.14.11",
     "npm-run-all": "4.0.2",
     "office-toolbox": "0.0.6",
-    "perfectionist": "2.4.0",
-    "postcss-loader": "1.3.3",
-    "precss": "1.4.0",
-    "raw-loader": "0.5.1",
+    "perfectionist": "^2.4.0",
+    "postcss-loader": "^2.1.3",
+    "precss": "^3.1.2",
+    "raw-loader": "^0.5.1",
     "rimraf": "2.6.1",
-    "sass-loader": "6.0.3",
+    "sass-loader": "^6.0.7",
     "shelljs": "0.7.7",
-    "style-loader": "0.16.1",
+    "style-loader": "^0.20.3",
```

Comment:
Someday you'll have to explain to me the usage of "^" -- in particular, the safety thereof (and also, how does that work from a complience/OSS perspective)